docs/ZendeskMultiObjects-batchsource.md

```diff
@@ -3,19 +3,13 @@
 
 Description
 -----------
-This source reads multiple objects from Zendesk. It extracts reportable data from the multiple Zendesk objects. 
-The Zendesk Multi Object Batch Source plugin enables bulk data extraction from Zendesk. 
-You can configure and execute bulk data transfers from Zendesk without any coding.
-
-Examples of objects are Article Comments, Post Comments, Requests Comments, Ticket Comments,
-Groups, Organizations, Satisfaction Ratings, Tags, Ticket Fields,
-Ticket Metrics, Ticket Metric Events, Tickets, Users.
-Tags object lists the 500 most popular tags in the last 60 days.
-
-The data which should be read is specified using object list and filters for those objects.
+The Zendesk Multi Object Batch Source plugin provides seamless extraction of reportable data from multiple objects 
+within Zendesk. By configuring and executing bulk data transfers, this source reads and extracts information from 
+Zendesk without the need for any coding.
 
 In addition, for each Object that will be read, this plugin will set pipeline arguments where the key is 
-'multisink.[ObjectName]' and the value is the schema of the Object.
+'multisink.[ObjectName]' and the value is the schema of the Object. It is designed to be consistently compatible with
+multiSink plugins.
```

Comment:
Add a blank line in between and just write This is only compatible with MultiSink plugins as a Sink.

Reply:
done